### PR TITLE
feat:Repair description disabled

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

Repair description disabled

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@wzh425 PTAL

File: src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/Components/AddSchedulerResourceFiles.razor [hunk 1]
```diff
@@ -33,6 +33,6 @@
             </div>
         </div>
         <STextField @bind-Value="Model.Version" Label="@T(nameof(Model.Version))" Class="mt-9" />
-        <STextarea @bind-Value="Model.Description" Label="@T(nameof(Model.Description))" Class="mt-9" HideDetails="@("auto")" Outlined />
+        <STextarea @bind-Value="Model.Description" Label="@T(nameof(Model.Description))" Class="mt-9" HideDetails="@("auto")" Disabled="Model.EnableDescription" Outlined />
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/Components/AddSchedulerResourceFiles.razor:36]: What's the use of this

File: src/Contracts/Masa.Scheduler.Contracts.Server/Dtos/SchedulerResourceDto.cs [hunk 2]
```diff
@@ -20,5 +20,7 @@ public class SchedulerResourceDto
     public DateTimeOffset UploadTime { get; set; }
 
     public Guid Creator { get; set; }
+
```
Comment (doddgu) [inline src/Contracts/Masa.Scheduler.Contracts.Server/Dtos/SchedulerResourceDto.cs:23]: Remove tha blank line.

**Review by wzh425 — APPROVED** [review]
LGTM

**Pull request merged**